### PR TITLE
tslint: add page

### PR DESCRIPTION
**Pull request by mstruebing**
----
see: https://github.com/palantir/tslint

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/tslint.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Lint on files:
+
+`tslint {{filename}}.js {{filename1}}.js`
+
+- Fix lint issues:
+
+`tslint --fix`
+
+- Lint with config in project root:
```
Comment (sbrl) [inline pages/common/tslint.md]: Is this a config file in a specific directory, or a specific config file?

Comment (mstruebing) [reply pages/common/tslint.md]: The config `tslint.json` needs to be in the project root path. Similiar to others `--config {{path/to/config}}` but there you need to specify the root path where the config is located.

File: pages/common/tslint.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# tslint
+
+> A pluggable linting utility for TypeScript.
+
+- Create tslint config:
+
+`tslint --init`
+
+- Lint on files:
```
Comment (agnivade) [inline pages/common/tslint.md]: Let's make the same change as `xo` and this should be good to go.

Lint on a given set of files.

File: pages/common/tslint.md (hunk 1 above)
Comment (sbrl) [reply pages/common/tslint.md]: Ah, I see. Perhaps this will read better then?

```suggestion
- Lint with the config file in the project root:
```

**Pull request merged**